index.html

```diff
@@ -183,5 +194,79 @@ <h3>Skills</h3>
         </div>
       </div>
     </section>
+    <!-- contactme -hana & henry-->
+    <section class="contactme">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        action="https://formspree.io/f/maykropo"
+        method="post"
+        class="contact-form"
+      >
+        <div class="firstlastname">
+          <input
+            class="form-element"
+            id="firstname"
+            type="text"
+            name="name"
+            placeholder="First Name"
+            maxlength="30"
+            required
+          />
+          <input
+            class="form-element"
+            id="secondname"
+            type="text"
+            name="name"
+            placeholder="Last Name"
+            maxlength="30"
+            required
+          />
```

Comment:
- [ ] In the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) it is stated that we should use a single name input regardless of the Figma template. Kindly remove these "First name" and "Last name" inputs and use only "Full name" for both mobile and desktop versions of the page. 🤝️

- **Requirement**
![Screenshot from 2022-12-08 22-17-19](https://user-images.githubusercontent.com/17243325/206550709-f46316e9-948d-47e3-8f6e-b6162bc757b0.png)

- **Your Design**
![Screenshot from 2022-12-08 22-14-23](https://user-images.githubusercontent.com/17243325/206550792-d7c81ba7-cc28-43a4-8466-7811ddadd74a.png)